openedx/core/djangoapps/plugin_api/views.py

```diff
@@ -82,6 +82,8 @@ def render_to_standalone_html(self, request, fragment, **kwargs):
         """
         Renders this fragment to HTML for a standalone page.
         """
+        if not fragment:
```

Comment:
@andy-armstrong You can leave this code as it was.  I meant that one could code this such that there is a fragment that renders nothing, which would be different than not having a fragment, but maybe there isn't a benefit to this.

Once you make a decision, could you documenting in the Docstring what happens when fragment is None?